### ISSUE TITLE
Update chroma quantization matrices

### ISSUE THREAD

**Issue by negge** [xiph/daala#8]
We should be quantizing chroma less coarsely than luma at low bit rates.  At the low end of the range, the quantization chroma should be a factor of two over luma.  At high bit rates we should be quantizing chroma more heavily (Cb more heavily than Cr).

**Comment by barrbrain** [comment]
I experimented with subjectively optimizing the chroma quantization curve.  I examined AB results at 0.1, 0.04 and 0.02 bpp for all of subset1. At each bpp I conducted a binary search on ratio between luma and chroma coded_quantizer. I fitted a quadratic curve through these points to give a single numeric parameter. I noticed that this parameter was very close to 1/4, so I chose that value for the final version of my function.
To demonstrate the effects, I animated all values of -v for each image in subset1. I have uploaded Clovisfest and the rest will come soon: [Chroma Opt](https://drive.google.com/folderview?id=0B_6vZm7qZLK6OUZvZFlBQ2tOaTQ) (drive.google.com)
